.gitignore

```diff
@@ -1 +1,3 @@
-./idea
```

Comment:
Maybe consider using some existing `.gitignore` files for python like [this one](https://github.com/github/gitignore/blob/main/Python.gitignore).

Reply:
Done.